lib/terraform/command_options.rb

```diff
@@ -56,6 +56,14 @@ def state=(value)
       store key: 'state', value: value
     end
 
+    def state_out
+      fetch key: 'state-out'
+    end
+
+    def state_out=(value)
```

Comment:
I think it makes more sense to have this method named `state_out_(store|set|equal)` rather than a literal `=`